doc/meson.build

```diff
@@ -31,25 +31,27 @@ glib_docpath = join_paths(glib_prefix, 'share', 'gtk-doc', 'html')
 
 docpath = join_paths(xapian_glib_datadir, 'gtk-doc', 'html')
 
-gnome.gtkdoc('xapian-glib',
```

Comment:
Nothing changed here except the formatting, is that intended?

Reply:
Yes; I kept the coding style consistent.

Reply:
Now that you mention it, no: it wasn't intended. Just fixedit.